hack/flags.sh

```diff
@@ -59,7 +54,7 @@ parse_flags() {
     esac
   done
 
-  if [[ -z ${KCP_KUBECONFIG} ]] || [[ -z ${KCP_KUBECONFIG} ]]
+  if [[ -z ${KCP_KUBECONFIG} ]] || [[ -z ${CLUSTER_KUBECONFIG} ]]
   then
     echo "ERROR: Both parameters --kcp-kubeconfig and --cluster-kubeconfig are mandatory" >&2
     exit 1
```

Comment:
btw, since you introduced the variables in `preview.env` file, wouldn't it be easier to use only one source of the values?
```suggestion
  if echo ${MODE} | grep -q preview
  then
    if [[ -n ${KCP_KUBECONFIG}${CLUSTER_KUBECONFIG} ]]
    then
      echo "ERROR: You cannot use the parameter --kcp-kubeconfig nor --cluster-kubeconfig in preview mode - use only the './hack/preview.env' file" >&2
      exit 1
    elif [[ -f ${ROOT}/hack/preview.env ]]
    then
      echo "Loading environment variables from ${ROOT}/hack/preview.env"
      source ${ROOT}/hack/preview.env
    else
      echo "ERROR: No ${ROOT}/hack/preview.env was found" >&2
      exit 1
    fi
  elif [[ -z ${KCP_KUBECONFIG} ]] || [[ -z ${CLUSTER_KUBECONFIG} ]]
  then
    echo "ERROR: Both parameters --kcp-kubeconfig and --cluster-kubeconfig are mandatory" >&2
    exit 1
```
I know that the --mode is loaded only in some scripts, but it doesn't really matter, does it? This part fill work anyway